openapi.yaml

```diff
@@ -10615,6 +10618,300 @@ components:
+          items:
+            $ref: '#/components/schemas/WAContentButton'
+      required:
+        - body
+      x-examples:
+        Example 1:
+          header:
+            image:
+              link: 'http://placekitten.com/200/300'
+          body:
+            - text: John
+            - text: ACTIVE
+            - text: '+619702873117'
+          button:
+            - index: '0'
+              sub_type: quick_reply
+              payload: Yes-Button-Payload
+            - index: '1'
+              sub_type: quick_reply
+              payload: No-Button-Payload
+    WAContentHeader:
+      title: WAContentHeader
+      x-stoplight:
+        id: hevlc9qwqt4y3
+      type: object
+      x-examples:
+        Image Example:
+          header:
+            image:
+              link: 'http://placekitten.com/200/300'
+        Video Example:
+          video:
+            link: 'https://www.learningcontainer.com/wp-content/uploads/2020/05/sample-mp4-file.mp4'
+        Document Example:
+          document:
+            link: 'https://s3-eu-west-1.amazonaws.com/hughhopkins.co.uk/sample_media/Sf+flight+itinerary.pdf'
+        Location Example:
+          location:
+            longitude: -122.425332
+            latitude: 37.758056
+            name: Facebook HQ
+            address: '1 Hacker Way, Menlo Park, CA 94025'
+      description: 'The header of your message can include one type of media content e.g. image, video, document and location.'
+      properties:
+        image:
+          title: Object
+          type: object
+          description: Link to image location
+        video:
+          title: Object
+          type: object
+          description: Link to video location
+        document:
+          title: Object
+          type: object
+          description: Link to document location
+        location:
+          title: Object
+          type: object
+          description: '“Longitude”, "latitude", "name", and "address"'
+    WAContentButton:
+      title: WAContentButton
+      x-stoplight:
+        id: 8rugv1dziyhzp
+      type: object
+      description: |-
+        The contents of your dynamic button.
+
+        If the button subtype is “url” you will need the following parameters:
+        “Index”, “sub_type”, “text”
+
+        If the button subtype is a “quick reply”, you will need the following parameters:
+        “Index”, “sub_type”, “payload
+      x-examples:
+        Example 1:
+          button:
+            - index: '0'
+              sub_type: quick_reply
+              payload: Yes-Button-Payload
+            - index: '1'
+              sub_type: url
+              payload: 'https://google.com'
+      properties:
+        index:
+          type: string
+          description: The index value of the button this object represents
+          example: '0'
+        sub_type:
+          type: string
+          description: The type of dynamic button i.e. "quick reply" or "url"
+          example: 'url'
```

Comment:
We can also use an `enum` for this property:

```
type: string
enum:
  - quick_reply
  - url
```

Reply:
Added enum to button subtype